src/test/java/org/medicmobile/webapp/mobile/RequestLocationPermissionActivityTest.java

```diff
@@ -49,9 +41,10 @@ public void setup() {
 
 	@Test
 	public void onClickAllow_withPermissionGranted_setResolveOk() {
-		try(MockedStatic<MedicLog> medicLogMock = mockStatic(MedicLog.class)) {
-			ActivityScenario<RequestLocationPermissionActivity> scenario = scenarioRule.getScenario();
-
+		try(
+			MockedStatic<MedicLog> medicLogMock = mockStatic(MedicLog.class);
+			ActivityScenario<RequestLocationPermissionActivity> scenario = ActivityScenario.launchActivityForResult(RequestLocationPermissionActivity.class)
```

Comment:
Something changed in the new version of Robolectric where it did not want you to call `getResult` on scenarios that were created from the Rule. Instead, the scenario  needed to be created with this `launchActivityForResult` call.